nix/sources.json

```diff
@@ -11,6 +11,19 @@
         "url": "https://github.com/tweag/gomod2nix/archive/67f22dd738d092c6ba88e420350ada0ed4992ae8.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "gravity-bridge": {
+        "branch": "main",
+        "description": "A CosmosSDK application for moving assets on and off of EVM based, POW chains",
```

Comment:
gravity bridge supports only Cosmos <-> Ethereum at the moment

Reply:
I think it means transfer to and from EVM chain?